packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -349,6 +350,49 @@ class IosProject extends XcodeBasedProject {
   // The string starts with `applinks:` and ignores the query param which starts with `?`.
   static final RegExp _associatedDomainPattern = RegExp(r'^applinks:([^?]+)');
 
+  static const String _lldbInitTemplate = '''
+#
+# Generated file, do not edit.
+#
+
+command script import --relative-to-command-file lldb_helper.py
+''';
+
+  static const String _lldbPythonHelperTemplate = r'''
+#
```

Comment:
I wonder if this should just live as a file in Flutter SDK repo? It would be good to be able to update its contents as necessary (e.g. when Flutter SDK - Dart VM changes). Currently each user project will get its own copy. 

Maybe we could do the following:

- Add `flutter_lldb_helper.py` somewhere in Flutter SDK.
- Before project is run we overwrite `ios/flutter_lldbinit` with contents which use absolute path to Flutter SDK: e.g. `command script import /full/path/to/flutter/sdk/flutter_lldb_helper.py`

WDYT?

Reply:
Ah! I was missing this piece of information. If that's the case then current approach totally LGTM.